src/styles/card.scss

```diff
@@ -0,0 +1,81 @@
+.card {
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  padding: 32px 16px 16px;
+  box-sizing: border-box;
+  transition:
+    background-color 300ms,
+    color 300ms,
+    border 300ms;
```

Comment:
According to the code style rules, when properties have multiple values, each value should be on a new line with an extra level of indentation. This helps to keep the code clean and readable.